themes/SuiteP/css/suitep-base/search.scss

```diff
@@ -1,45 +1,3 @@
-#search-wrapper-form {
-  $border-radius: 20px;
-  $margin: 60px;
-
-  margin-top: $margin;
-  margin-bottom: $margin;
-
-  label {
-    position: absolute;
-    top: -17px;
-    left: 12px;
-  }
-
-  .msfcol {
-    position: relative;
-    margin: -3px;
-  }
-
-  & input, & select {
-    width: 100%;
-    height: 40px;
-    border-radius: 0;
-    background: $color-44;
-    border: 1px solid $color-34;
```

Comment:
select dropdown has a border on classic views (for e.g edit view), can you please re-add it?
![image](https://user-images.githubusercontent.com/13831178/49096534-a7c19a80-f262-11e8-90d0-fc85571c5788.png)
=> like this:
![image](https://user-images.githubusercontent.com/13831178/49096558-b1e39900-f262-11e8-94e2-277bc7731ab8.png)